### DISCUSSION TITLE
Support of HA_engine_address to support username & password (and TLS)

### DISCUSSION THREAD

**Discussion by jmjiii** [grafana/grafana#66320] [Platform]
We have a need to support modern auth controls to Redis.  Looking over the code, it does not support a redis URL.  
Looking at centrifuge, the underlying package supports it, but the core ansible.ini & ping code does not.  It would be very useful to be able to use fully configuration redis URL or connection string

[live]
ha_engine = redis
ha_engine_address = redis://username:passwd@xyz.com&ssl=true

or
xyz.redis.cache.windows.net:6379,password=wombat=,ssl=True,abortConnect=False

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). We are migrating this discussion to an issue and closing the discussion. The issue is #82715. Feel free to continue the discussion around this there. Thank you!
